libs/portal/pages/challenges.tsx

```
@@ -0,0 +1,115 @@
+        <AccordionContent>
+          <div className="flex flex-col gap-2">
+            {challenge.challengeBlock
+            && challenge.challengeBlock.map(block =>
+              block.blockType === 'info'
+                ? (
+                    <ChallangeBlockBullet key={block.id} blockData={block} />
+                  )
+                : (
+                    <ChallangeBlockButtons key={block.id} blockData={block} />
+                  ),
+            )}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)